src/data-requests/slack.js

```diff
@@ -35,7 +39,7 @@ export const submitSlackDataRequest = async (
     pediatric_cancer: pediatricCancer,
     query
   } = requestValues
-  const host = new URL(req.url).hostname
+  const host = new URL(`http://${req.headers.host}${req.url}`).hostname
```

Comment:
update this to have valid origins for prod and staging, then have a default which should be staging in this case because 

add a note that for security to test changes for the picture you have to merge into staging

Reply:
I've set the origin url to either prod or staging (by default) based on the value of `req.headers.origin`, and added the staging url to `config/links` (at 9e9f8372464e97c621214ac16c92e60f3e93c093), thank you David!